pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -1066,3 +1069,59 @@ type PuppetDBSDConfig struct {
 	// +optional
 	EnableHTTP2 *bool `json:"enableHTTP2,omitempty"`
 }
+
+type LightSailSDConfig struct {
+	// The AWS region
+	// TODO: seems like an empty string is valid for the Region, should we make it Required
+	// and let it be, or leave it optional? considering that Prometheus does not
+	// check complain if it is nil.
+	// +optional
+	Region *string `json:"region,omitempty"`
+	// AccessKey is the AWS API key.
+	// +optional
+	AccessKey *corev1.SecretKeySelector `json:"accessKey,omitempty"`
+	// SecretKey is the AWS API secret.
+	// +optional
+	SecretKey *corev1.SecretKeySelector `json:"secretKey,omitempty"`
+	// AWS Role ARN, an alternative to using AWS API keys.
+	// TODO: Should we validate against "^arn:aws:connect:\w+(?:-\w+)+:\d{12}:instance\/[A-Za-z0-9]+(?:-[A-Za-z0-9]+)+$"
```

Comment:
Not sure about this one. Maybe tackle it later with AWS SD?

Reply:
I will leave the TODO, just in case. We can come back to it while refactoring.

Reply:
The `TODO: ...` line appears in the API docs too which isn't very clean. I'll remove it for now.